pinot-tools/src/main/java/org/apache/pinot/tools/Command.java

```diff
@@ -18,12 +18,21 @@
  */
 package org.apache.pinot.tools;
 
+import java.util.concurrent.Callable;
+
+
 /**
  * Interface class for pinot-admin commands.
  *
  *
  */
-public interface Command {
+public interface Command extends Callable<Integer> {
+
+  default Integer call() throws Exception {
+    // run execute() and returns 0 if success otherwise return -1.
+    return execute() ? 0 : -1;
```

Comment:
I think the cmd return 0 if success otherwise return 1.

Reply:
i think any non-zero return code indicates a failure. but yes I can keep it consistent if previously it is using `1` as failure return code.

Reply:
```suggestion
    return execute() ? 0 : 1;
```